### DISCUSSION TITLE
Website requirments

### DISCUSSION THREAD

**Discussion by Razor421** [ASEAN-Build-The-Earth/AseanBTE-Website#2] [Q&A] [answered]
I am currently trying to build our website in HTML. I was wondering what are the requirements for this website are.

**Comment by StoneMcYT** [comment, marked as answer]
Something like this i suppose. You can use any colour scheme etc
but main points are 

-  Page dedeicated for admin updates/posts
- Page for current staff and devs
- Page for build locations at the moment
- Page for insta links contacts twitter etc

https://aseanbte.webflow.io/
